docs/working-with-projects/project-configuration/create-robot-accounts.md

```diff
@@ -13,33 +13,33 @@ You can create robot accounts to run automated operations. Robot accounts have t
 1. Log in to the Harbor interface with an account that has at least project administrator privileges.
 1. Go to **Projects**, select a project, and select **Robot Accounts**.
 
-    ![Robot accounts](../../../img/add-robot-account.png)
+    ![Robot accounts](../../img/add-robot-account.png)
```

Comment:
‘../../../' might be correct, it's always one level more than correct directory structure, please make sure

Reply:
same for all the images in the rest of this document